docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -79,6 +79,14 @@ gives a convenient interface for the RPC methods.
   - [slotSubscribe](jsonrpc-api.md#slotsubscribe)
   - [slotUnsubscribe](jsonrpc-api.md#slotunsubscribe)
 
+### Deprecated Methods
```

Comment:
For each of the deprecated methods, how about adding a statement indicating the release that we expect to remove them in